README.md

```diff
@@ -10,6 +10,7 @@
 🇹🇼
 🇺🇦
 🇦🇿
+PH/PHL
```

Comment:
Why not 🇵🇭 ? (I literally googled "Philippines flag emoji", so if there is some deeper political situation I'm unaware of, just let me know)